From 74fd5098b1c735cf3e86b876d9f821491a572f37 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mariusz=20Bia=C5=82o=C5=84czyk?= <manio@skyboo.net>
Date: Sat, 30 Dec 2023 17:31:15 +0100
Subject: [PATCH] Fix compilation: update random.h
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

error:
```
In file included from /usr/src/freeserf-git/src/freeserf/src/random.cc:22:
/usr/src/freeserf-git/src/freeserf/src/random.h:29:3: error: ‘uint16_t’ does not name a type
   29 |   uint16_t state[3];
      |   ^~~~~~~~
/usr/src/freeserf-git/src/freeserf/src/random.h:26:1: note: ‘uint16_t’ is defined in header ‘<cstdint>’; did you forget to ‘#include <cstdint>’?
   25 | #include <string>
  +++ |+#include <cstdint>
   26 |
```
---
 src/random.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/random.h b/src/random.h
index 9b548d47b..1e6e8ec90 100644
--- a/src/random.h
+++ b/src/random.h
@@ -23,6 +23,7 @@
 #define SRC_RANDOM_H_
 
 #include <string>
+#include <cstdint>
 
 class Random {
  protected:
